src/APINavData.php

```diff
@@ -12,8 +12,11 @@
 
 final class APINavData {
   private APIIndex $index;
+  private string $product;
```

Comment:
Is the `APIProduct` enum usable here? It be best to keep it as strictly typed as possible until the last momenet, casting to string if necessary when creating the URL - though AIUI it shouldn't need an explicit cast.

Reply:
really good point -- resolves fine. merging in with that change.